e2e/pages/organizations/onc-acbs-atls/atls.spec.js

```diff
@@ -0,0 +1,126 @@
+import ToastComponent from '../../../components/toast/toast.po';
+
+let address; let hooks; let login; let page; let
+  toast;
+
+describe('the ONC-ATL Management page', () => {
+  const timestamp = (new Date()).getTime();
+  const websiteUrl = `http://www.example${timestamp}.com`;
+  const today = new Date().toLocaleDateString('en-US', { year: 'numeric', month: '2-digit', day: '2-digit' });
+  const atlAddress = {
+    address: `address${timestamp}`,
+    city: `city${timestamp}`,
+    state: `state${timestamp}`,
+    zip: `11111${timestamp}`,
+    country: `country${timestamp}`,
+  };
+
+  beforeEach(async () => {
+    browser.setWindowSize(1600, 1024); // demo of a bigger screen (esp. useful for screenshots)
+    browser.setWindowRect(0, 0, 1600, 1024); // not sure if both are required
+    page = new OrganizationPage();
+    hooks = new Hooks();
+    address = new AddressComponent();
+    login = new LoginComponent();
+    toast = new ToastComponent();
+    await hooks.open('#/organizations/onc-atls');
+  });
+
+  describe('when impersonating as an ATL', () => {
+    beforeEach(() => {
+      const userID = '41';
+      login.logIn('onc');
+      login.logoutButton.waitForDisplayed();
+      hooks.open('#/users');
+      page.impersonateUser(userID).scrollIntoView({ block: 'center', inline: 'center' });
+      page.impersonateUser(userID).click();
+      hooks.open('#/organizations/onc-atls');
+    });
+
+    afterEach(() => {
+      const atl = 'UL LLC';
+      page.organizationNameButton(atl).click();
+      page.organizationEditButton.click();
+      page.organizationName.setValue(atl);
+      page.saveOrganizationButton.click();
+      login.logOut();
+    });
+
+    it('should allow user to edit own ATL details', () => {
+      const atl = 'UL LLC';
+      const newAtlName = `${atl} - ${timestamp}`;
+      const organizationType = 'ATL';
+      const atlId = '1';
+      page.organizationNameButton(atl).click();
+      page.organizationEditButton.click();
+      page.organizationName.setValue(newAtlName);
+      page.organizationWebsite.setValue(websiteUrl);
+      address.set(atlAddress);
+      page.saveOrganizationButton.click();
+      toast.clearAllToast();
+      expect(page.generalInformation(organizationType, atlId).getText()).toContain(newAtlName);
+      expect(page.generalInformation(organizationType, atlId).getText()).toContain(websiteUrl);
+      expect(page.generalInformation(organizationType, atlId).getText()).toContain(atlAddress.address);
+      expect(page.generalInformation(organizationType, atlId).getText()).toContain(atlAddress.city);
+      expect(page.generalInformation(organizationType, atlId).getText()).toContain(atlAddress.state);
+      expect(page.generalInformation(organizationType, atlId).getText()).toContain(atlAddress.zip);
+      expect(page.generalInformation(organizationType, atlId).getText()).toContain(atlAddress.country);
+    });
+  });
+  describe('when logged in as ONC', () => {
```

Comment:
```suggestion
  });
  
  describe('when logged in as ONC', () => {
```

I think we try to separate `describe` and `it` blocks with blank lines, though I haven't found an eslint rule that enforces that